Add banner and logos

miukimiu opened the pull request:
This PR adds a banner to the `Readme` file.

## Logo usage

![pgroll-logo-preview](https://github.com/xataio/pgroll/assets/2750668/19461519-7535-4062-a42a-0813e50eb880)

snide reviewed (approved):
LGTM from a design side. Only recommendation might be to provide SVGs as well in case anyone needs them.

miukimiu commented:
@snide I added the SVGs and removed the PNGs. I also created a [markdown brand kit page](https://github.com/xataio/pgroll/blob/bedca0d0c0d20990778bed8f48885986603817e8/brand-kit/README.md).
